### ISSUE TITLE
How to set infowindow's maxWidth?

### ISSUE THREAD

**Issue by akosijiji** [bradcornford/Googlmapper#78]
Hi @bradcornford 

Do you have any option to set infowindow's maxWidth? I can't find any it in the documentation. If you could provide some it would be helpful.

Thanks!

**Comment by brad** [comment]
I am @brad ;-)

**Comment by akosijiji** [comment]
@brad sorry wrong mention :D

**Comment by bradcornford** [comment]
Hi there,

There isn't currently an option to set the maxWidth option for information windows. I will look to implement this soon.

**Comment by bradcornford** [comment]
You do have an option to wrap you content in a div, with a set max width, such as:

```php

Mapper::informationWindow($space->latitude, $space->longitude,
'<div style="max-width: 200px;">Some long content............</div>', ['open' => true, 'markers' => ['title' => 'Title']]);



```

**Comment by bradcornford** [comment]
I have now implemented this in versions v2.15.0/v1.11.0.

It's usage would be as follows:

```php

Mapper::informationWindow(53.4, -1.5, 'Content', ['maxWidth' => 200]);

```
